spec/rubocop/cop/rspec/capybara/specific_matcher_spec.rb

```diff
@@ -0,0 +1,335 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::RSpec::Capybara::SpecificMatcher do
+  it 'does not register an offense for `have_css` when elements with' \
+    'no matcher specified in detail' do
+    expect_no_offenses(<<-RUBY)
+      expect(page).to have_css('article')
+      expect(page).to have_css('body')
+      expect(page).to have_css('tbody')
+    RUBY
+  end
+
+  it 'does not register an offense for `have_css` when elements with' \
+    'not element' do
+    expect_no_offenses(<<-RUBY)
+      expect(page).to have_css('.a')
+      expect(page).to have_css('#button')
+      expect(page).to have_css('[table]')
+    RUBY
+  end
+
+  it 'does not register an offense for `have_no_css` when elements with' \
```

Comment:
Does it make sense to combine examples of different matchers (`have_css`, `have_no_css`, `has_css`, `has_no_css`) into one example?

Reply:
spec structure was creating redundancy, so I reviewed the structure and organized the tests. What do you think?